auto_nag/scripts/workflow/no_severity.py

```diff
@@ -236,6 +238,23 @@ def get_bz_params(self, date):
                 }
             )
 
+        # TODO: the following code can be removed in 2023.
```

Comment:
Could you file an issue to track the removal? Or did you want to use #1592 to track it?

Reply:
I filed a new issue: https://github.com/mozilla/relman-auto-nag/issues/1596